docs/guides/capacity-planning.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Capacity Planning"
+linkTitle: "Capacity Planning"
+weight: 104
+slug: capacity-planning
+---
+
+
+You will want to estimate how many nodes are required, how many of
+each component to run, and how much storage space will be required.
+In practice, these will vary greatly depending on the metrics being
+sent to Cortex.
+
+Some key parameters are:
+
+ 1. The number of active series. If you have Prometheus already you
+ can query `prometheus_tsdb_head_series` to see this number.
+ 2. Sampling rate, e.g. a new sample for each series every 15
```

Comment:
The default scraping internal, it's actually 1 minute. I think the examples around the documentation tend to use 15s.